controlplane/kubeadm/internal/workload_cluster.go

```diff
@@ -120,6 +120,7 @@ type WorkloadCluster interface {
 	RemoveNodeFromKubeadmConfigMap(ctx context.Context, nodeName string, version semver.Version) error
 	ForwardEtcdLeadership(ctx context.Context, machine *clusterv1.Machine, leaderCandidate *clusterv1.Machine) error
 	AllowBootstrapTokensToGetNodes(ctx context.Context) error
+	AddClusterAdminRoleBinding(ctx context.Context, version semver.Version) error
```

Comment:
`AllowBootstrapTokensToGetNodes` starts will `Allow`, while this name starts with `Add`. the majority of methods start with `Reconcile` but in the background they all call `EnsureResource`.

name could be:
- `ReconcileClusterAdminRoleBinding`
- `AllowClusterAdminPermissions`.

to avoid adding another verb prefix.